etc/genome/spec/lsf_resource_annotate_variants.yaml

```diff
@@ -0,0 +1,3 @@
+---
+default_value: "-M 4000000 -R 'select[mem>=4000 && tmp>=2000] rusage[mem=4000,tmp=2000]'"
```

Comment:
I believe our code converts both to whichever one we're using these days.  I've heard discussion on changing which one that is, but I don't think there's been recent movement on that.

FWIW I copied this resource string from another config verbatim.